core/src/main/java/org/apache/accumulo/core/iterators/system/LocalityGroupIterator.java

```diff
@@ -16,23 +16,28 @@
  */
 package org.apache.accumulo.core.iterators.system;
 
+import com.google.common.collect.ImmutableSet;
```

Comment:
Oh, oops. I thought we had one, but I could see it just being eclipse's default. At least one other project I'm a part of uses https://google.github.io/styleguide/javaguide.html#s3.3-import-statements which I've grown to like.

Anyways, this is becoming quite the tangent. Thanks for changing it back, Ivan.